simple_certmanager/models.py

```diff
@@ -3,11 +3,16 @@
 from django.db import models
 from django.utils.translation import gettext, gettext_lazy as _
 
+from cryptography import x509
+from cryptography.hazmat.backends import default_backend
+from cryptography.hazmat.primitives import hashes, serialization
+from cryptography.hazmat.primitives.asymmetric import rsa
 from cryptography.hazmat.primitives.asymmetric.types import PrivateKeyTypes
 from cryptography.x509 import (
     Certificate as CryptographyCertificate,
     load_pem_x509_certificate,
 )
+from cryptography.x509.oid import NameOID
```

Comment:
```suggestion
from cryptography.x509 import NameOID
```
this is the public export - the `.oid` module is not picked up by (Pyright) type checker